File: Java/threedprinter.java
/*
 *	Problem: 3D Printed Statues
 */

import java.util.Scanner;
import java.lang.Math;

public class threedprinter {
	
	/*-----main-----*/
	public static void main(String[] args) {
		Scanner sc = new Scanner(System.in);
		int numStatues = sc.nextInt();
		
		double numDays = (Math.log(numStatues) / Math.log(2));
		int minNumDays = (int) Math.ceil(numDays);
		minNumDays++;
		
		System.out.println(minNumDays);
	} //end main
	
} // end class

File: JavaScript/threedprinter.js
/*
 *	Problem: "3D Printed Statues"
 */

const readline = require('readline');

const rl = readline.createInterface({
 input: process.stdin,
 output: process.stdout
});

rl.on('line', (line) => {
 var numStatues = parseInt(line);

	var numDays = Math.log2(numStatues);
	var minNumDays = Math.ceil(numDays);
	minNumDays++;

	console.log(minNumDays);
});